libcloud/common/base.py

```diff
@@ -278,12 +278,15 @@ def response(self):
         if not self._response:
             response = self.connection.connection.getresponse()
             self._response = HttpLibResponseProxy(response)
-            self.body = response.content
             if not self.success():
                 self.parse_error()
         return self._response
 
     @property
+    def body(self):
```

Comment:
Would be good to add a comment on why we do this.

And a test case would also be great, besides that, LGTM.

Reply:
Ah, right, will do.